src/Components/Officers.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import "../CSS/Officers.css";
+import ProfileCard from "./ProfileCard";
+
+const Officers = () => {
+  const [selectedYear, setSelectedYear] = useState(new Date().getFullYear());
+  const [teamMembers, setTeamMembers] = useState([]);
+
+  useEffect(() => {
+    const loadTeamMembers = async () => {
+      try {
+        const members = await import(`../Data/officers${selectedYear}.json`);
+        setTeamMembers(members.default);
+      } catch (error) {
+        console.error("Error loading team members:", error);
+        setTeamMembers([]);
+      }
+    };
+
+    loadTeamMembers();
+  }, [selectedYear]);
+
+  const handleYearChange = (year) => {
+    setSelectedYear(year);
+  };
+
+  return (
+    <div>
+      <div>
+        <div className="about-message-section">
+          <h1>Our Officers</h1>
+        </div>
+        <div className="Officer-content">
+          <nav className="officer-sidebar">
+            <h2>Year</h2>
+            <ul>
+              {[2024, 2023, 2022, 2021].map((year) => (
```

Comment:
Maybe we can not hardcode years, can check if after september then add the next year? Starting with 2021.